specification/BUILDING.md

```diff
@@ -0,0 +1,110 @@
+<!-- omit in toc -->
```

Comment:
Are `PropertiesReference_3dtiles.adoc` and `PropertiesReference_3dtiles_schema.adoc` auto-generated from wetzel? And will those instructions eventually go in here?

Reply:
These files indeed are the current output from (my local version of) wetzel. Similar files are generated by wetzel in its original `main` state, except for that it cannot properly produce them for 3D Tiles, of course.

There is a note at the top of `BUILDING.md` saying that the process of generating them with wetzel will be documented in the future (and maybe wrapped into some Docker container when necessary).

Reply:
Added instructitions for generating the properties reference to the build instructions. 
(If that docker thing is tackled at one point, this will be added accordingly)